config/locales/en/helpers.yml

```diff
@@ -195,6 +195,8 @@ en:
       steps_conviction_conviction_bail_days_form:
         conviction_bail_days: This is decided by the court. If you do not know the exact number of days you can enter an approximate number or leave this blank, but it might affect the accuracy of your result.
       steps_conviction_conviction_subtype_form:
+        adult_motoring_html: A fixed penalty notice (FPN) with no penalty points is a fine for a minor driving offence. It will not appear on your criminal record unless a court gives you a conviction because of one.
```

Comment:
You need to declare also the default key, otherwise when selecting other convictions, not motoring:

<img width="744" alt="Screenshot 2021-03-04 at 16 32 39" src="https://user-images.githubusercontent.com/687910/109996647-54730800-7d07-11eb-88b1-82a64fd6c720.png">